test/com/facebook/buck/jvm/kotlin/BUCK

```diff
@@ -137,8 +137,10 @@ java_library(
     ],
 )
 
-standard_java_test(
+java_test(
```

Comment:
this change should be reverted since we don't have a faux_kotlin_home directory anymore